zebra-chain/src/lib.rs

```diff
@@ -27,6 +27,21 @@ pub use redjubjub;
 #[cfg(test)]
 use proptest_derive::Arbitrary;
 
+use serde_helpers::*;
+
+/// The size of an Equihash solution in bytes (always 1344).
+const EQUIHASH_SOLUTION_SIZE: usize = 1344;
+
+mod serde_helpers {
+    use super::EQUIHASH_SOLUTION_SIZE;
+    use serde_big_array::big_array;
```

Comment:
I'm fine with using `serde_big_array` if other people are, but I think we should use it consistently if we use it at all, there are manual Deserialize implementations for BCTV14 and Groth16 proofs that should be removed if we opt to use it.